README.md

```diff
@@ -11,140 +11,109 @@
 
 **Status: beta**
 
-When working on large code trees, it's common to have multiple code
-formatters run against it. And have one script that loops over all of them.
-`treefmt` makes that nicer.
+`treefmt` applies all the needed formatters to your project with one command line.
 
-- A unified CLI and output
-- Run all the formatters in parallel.
-- Cache which files have changed for super fast re-formatting.
+## Motivation
 
-Just type `treefmt` in any folder and it reformats the whole code tree.
+Before making contributions to any project, it’s common to get your code formatted according to the project’s standards. This task seems trivial from the first sight — you can simply set up the required language formatter in your IDE. But contributing to multiple projects requires more effort: you need to change the code formatter configs each time you switch between projects, or call formatters manually.
 
-## Design decisions
+Formatting requires less effort if a universal formatter for multiple languages is in place, which is also project-specific.
 
-We assume that the project code is checked into source control. Therefore, the
-default should be to write formatter changes back in place. Options like
-`--dry-run` are not needed; the source control is relied upon to revert or
-check for code changes.
+## About treefmt
 
-`treefmt` is responsible for traversing the file-system and mapping files to
-specific code formatters.
+`treefmt` runs all your formatters with one command. It’s is easy to configure and fast to execute.
 
-Only _one_ formatter per file. `treefmt` enforces that only one tool is
-executed per file. Guaranteeing two tools to produce idempotent outputs is
-quite tricky.
+[![asciicast](https://asciinema.org/a/cwtaWUTdBa8qCKJVp40bTwxf0.svg)](https://asciinema.org/a/cwtaWUTdBa8qCKJVp40bTwxf0)
+
+Its main features are:
+
+- Providing a unified CLI and output: You don’t need to remember which formatters are required for each project. Once you specify the formatters in the config file, you can trigger all of them with one command and get a standardized output.
+- Running all the formatters in parallel: A standard script loops over your folders and runs each formatter consequentially. In contrast, `treefmt` runs formatters in parallel. This way, the formatting job takes less time.
+- Caching the changed files: When formatters are run in a script, they process all the files they encounter, no matter whether the code has changed. This unnecessary work can be eliminated if only the changed files are formatted. `treefmt` caches the changed files and marks them for re-formatting.
+
+Just type `treefmt` in any folder to reformat the whole code tree. All in all, you get a fast and simple formatting solution. Find the project wiki [here](https://github.com/numtide/treefmt/wiki).
```

Comment:
```suggestion
Just type `treefmt` in any folder to reformat the whole code tree. All in all, you get a fast and simple formatting solution.
```
The transition in the sentence is a bit weird. The main use of the wiki is to share formatter formulas, so it could be introduced somewhere down the line?